xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -335,6 +335,20 @@
     ],
     "returns": "string"
   },
+  "Player.ViewMode": {
+    "type": "method",
+    "description": "ViewMode of VideoPlayer",
+    "transport": "Response",
+    "permission": "ControlPlayback",
+    "params": [
+      { "name": "viewmode",  "type": "integer", "description": "view mode numbers", "required": true },
+      { "name": "zoom", "type": "number", "required": "false", "default": 1.0, "description": "Zoom were 1.0 means 100%" },
+      { "name": "pixelratio", "type": "number", "required": "false", "default": 1.0, "description": "Pixel aspect ratio were 1.0 means square pixel" },
+      { "name": "verticalshift", "type": "number", "required": "false", "default": 0.0, "description": "Verticalshift 1.0 means shift to bottom" },
+      { "name": "stretch", "type": "boolean", "required": "false", "default": false, "description": "Flag to enable nonlinear stretch" }
```

Comment:
Can you change "stretch" to "nonlinearstretch" or "nonlinear"? Increasing the pixel ratio will linear-stretch the video so it would be nice to distinguish that in the name, and this external API is the place to break from internal API for clarity.

Reply:
done - changed to nonlinearstretch